src/Sylius/Bundle/AdminBundle/templates/customer/show/component/order_statistics.html.twig

```diff
@@ -1,6 +1,6 @@
-{% from '@SyliusAdmin/shared/helper/accordion.html.twig' import default as _accordion %}
-{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon as _icon %}
-{% from '@SyliusAdmin/shared/helper/money.html.twig' import format as _money %}
+{% from '@SyliusAdmin/shared/helper/accordion.html.twig' import default as accordion %}
```

Comment:
Maybe change the main macro into `accordion`? Using `default` for the name is weird